config/search/config.json

```diff
@@ -295,6 +295,70 @@
         }
       }
     },
+    {
+      "type": "pieces",
+      "on_path": "pieces",
+      "rdf_type": "https://data.vlaanderen.be/ns/dossier#Stuk",
+      "properties": {
+        "title": "http://purl.org/dc/terms/title",
+        "created": "http://purl.org/dc/terms/created",
+        "fileName": [
+          "http://www.w3.org/ns/prov#value",
+          "http://www.semanticdesktop.org/ontologies/2007/03/22/nfo#fileName"
+        ],
+        "documentType": [
+          "^https://data.vlaanderen.be/ns/dossier#Collectie.bestaatUit",
+          "http://purl.org/dc/terms/type",
+          "http://mu.semte.ch/vocabularies/core/uuid"
+        ],
+        "accessLevel": [
+          "http://data.vlaanderen.be/ns/besluitvorming#vertrouwelijkheidsniveau"
+        ],
+        "mandateeIds": [
+          "^http://data.vlaanderen.be/ns/besluitvorming#geagendeerdStuk",
+          "http://mu.semte.ch/vocabularies/ext/heeftBevoegdeVoorAgendapunt",
+          "http://data.vlaanderen.be/ns/mandaat#isBestuurlijkeAliasVan",
+          "http://mu.semte.ch/vocabularies/core/uuid"
+        ],
```

Comment:
This restricts all documents search results to documents of agendaitems.
Is this the intention or do we want to be able to search all documents across the board?
Perhaps we need to not have a minister filter in the documents search route

Reply:
Good point. I also don't think mu-search will allow us as-is to check for branching paths (i.e. gather all mandatees via paths to the agendapoint, via paths to the subcase, via paths to the publication, etc.).

Will double check with Hannes what the intention was.

Reply:
Unsure of how search works on this. Could we define multiple paths for the same doc?
1 doc can be connected to a subcase (via submission), to a different subcase (linked doc directly), agendaitem (directly on multiple versions), to a case (directly by sync-service), to a publicationFlow (if VIA MR)

Reply:
I think we could have `agendaitemMandateeIds`, `agendaitemLinkedMandateeIds`, `subcaseMandateeIds`, ... and then in the frontend form the query correctly so that the selected mandatees are found in one or more of those fields.

Reply:
Currently, the search results open in document preview.
We currently have no way in the frontend to go from that preview to another context (like the agendaitem or subcase doc view).

Reply:
> I think we could have `agendaitemMandateeIds`, `agendaitemLinkedMandateeIds`, `subcaseMandateeIds`, ... and then in the frontend form the query correctly so that the selected mandatees are found in one or more of those fields.

All of those params would end up in the address bar + all the id's.
Pretty sure we are going to hit a limit of address length at some point

Reply:
> > I think we could have `agendaitemMandateeIds`, `agendaitemLinkedMandateeIds`, `subcaseMandateeIds`, ... and then in the frontend form the query correctly so that the selected mandatees are found in one or more of those fields.
> 
> All of those params would end up in the address bar + all the id's. Pretty sure we are going to hit a limit of address length at some point

No I meant that the index will contain those fields, and then the request we send will have those fields specified as a filter (so something like `filter[:terms:agendaitemMandateeIds, subcaseMandateeIds, ...]=aaaa-bbbb-cccc-dddd`. The query param will just remain `ministers` with one flat list. So the user looks for documents linked to e.g. Jambon, no matter through which path.